docs/errors.md

```diff
@@ -36,3 +36,22 @@ for more information about how to configure the `AutoScalerConfig`.
 Note that the autoscaler will execute in the same Kubernetes pod as
 the Ray head, so you must have a Kubernetes node available which can
 accomodate the resources for the head PLUS the autoscaler.
+
+## cannot open shared object file
```

Comment:
Users probably won't run into this, and if they do they might naturally discover solution (1). But it's good to have this here just in case.